tests/sentry/integrations/github/test_webhooks.py

```diff
@@ -588,7 +591,7 @@ def test_creates_missing_repo(self, mock_metrics):
         assert repos[0].external_id == "35129377"
         assert repos[0].provider == "integrations:github"
         assert repos[0].name == "baxterthehacker/public-repo"
-        mock_metrics.incr.assert_called_with("github.webhook.repository_created")
+        mock_metrics.incr.assert_any_call("github.webhook.repository_created")
```

Comment:
these tests are now using `assert_any_call` because a `PullRequestEvent` webhook will create the missing repo. additionally if the PR is opened and the org has the open PR comment org option (on by default), we will also emit a `github.open_pr_comment.queue_task` metric as the last metric and `assert_called_with` checks the last metric. in the test the option is on by default